script/1_0_2current_en_script/Arcueid/Day 7/07_00_ARC07_1.txt

```diff
@@ -29,74 +29,75 @@ Perhaps it's just me, but her bow doesn't seem as sharp as usual.
 <Page9>
 "...I wonder if she's worried about me. However, this kind of thing really does happen all the time."
 <Page10>
-I appreciate the concern, but it's nothing to be fussed about.
-If there is anything to be concerned about,
+I appreciate the concern, but it's nothing to be worried about.
+If there's anything to be concerned about,
 it would be a serious wound like that, no matter how you look at it.
 <Page11>
 "...But using duct tape really doesn't sit right with me. It makes my blood run cold just by looking at it. The next time I see her, I'll―――"
 <Page12>
-I stop just before I was able to finish my sentence.
+I stop just before I'm able to finish my sentence.
 There is no 'next time'.
 We'll never see each other again.
 There's no way I would get involved with such a dangerous woman.
 <Page13>
-"...That's right. I decided that would be for the best."
+"...That's right. It's obviously for the best."//「……そうだ。それがいいに決まってる」
```

Comment:
Can you explain this change? Closest approximation I can find is the definition for [決まって](https://www.weblio.jp/content/%E6%B1%BA%E3%81%BE%E3%81%A3%E3%81%A6) but this isn't really a correct TL for that either (more like 'always' or 'certainly'). Moreover, this is 決まってる which is just present-continuous 決まる
> ［副］ある条件のもとでは必ずそうなるさま。いつも。必ず。「忙しいときに—人が来る」

Am I missing some obvious phrase? If not, I'd suggest reverting or going to "We decided it would be for the best"

Reply:
Reverted